src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  &__container {
+    display: flex;
+    width: widthCatalogContent($width-card, $margin-catd, 4);
```

Comment:
There is a typo in the variable name `$margin-catd`. It should be `$margin-card`.